webapp/loggedin/modules/home/surveyInfo/languagesEditor.js

```diff
@@ -13,10 +15,12 @@ const LanguagesEditor = props => {
 
   const selection = surveyLanguages.map(lang => ({ key: lang, value: getLanguageLabel(lang) }))
 
+  const { i18n } = useContext(AppContext)
+
   return (
     <div className="form-item">
 
-      <label className="form-label">Language(s)</label>
+      <label className="form-label">{i18n.t('languagesEditor.languages')}</label>
 
```

Comment:
are you always rendering this label in plural form? why not to create a plural key and pass a count?

Reply:
It makes sense to me to always have it plural as it makes it explicit that the user can insert more than one language, otherwise it would change to singular or plural while the user is adding/removing languages. What do you think?